docs/releases/README.md

```diff
@@ -0,0 +1,19 @@
+# Release Workflow for Ega Metadata Schemas
+In this document you will find the workflow for creating releases of the Ega Metadata Schemas GitHub project. It provides a step-by-step diagram to help illustrate the steps involved.
+
+## Workflow Diagram
+![Release Workflow Diagram](images/Release_workflow_diagram.svg)
+
+
+The above diagram illustrates the process for creating a new release of the EGA Metadata Schemas project.
```

Comment:
I'll add the original link in the end. There's also the option to have it exported as an URL, but it would need to be changed every time there's a new version of the images, so the original URL is fine:
- [Release diagram](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&title=EE-2412%20-%20JSON%20Schema%20release#R%3Cmxfile%20pages%3D%222%22%3E%3Cdiagram%20id%3D%22C5RBs43oDa-KdzZeNtuy%22%20name%3D%22Release%20workflow%22%3E7V1bd5u4Fv41XtM%2BxIv75TH3ZE6TaZJ2MunLWcTINhMMLmDH6a8fCRAGtI0J5mY37lqNkUGA9tbWt68aiKez1aVnzKc3ronsgcCZq4F4NhAEXuZ4%2FIe0vEUtiqBHDRPPMuOT1g0P1i8UN3Jx68IykZ85MXBdO7Dm2caR6zhoFGTaDM9zX7OnjV07e9e5MUFMw8PIsNnWR8sMplGrJqjr9itkTab0zrwSv9%2FMoCfHb%2BJPDdN9TTWJ5wPx1HPdIPo2W50imwweHZfH67dH%2B8uLcvnnnf%2FT%2BH7yv2%2B3fx9FnV2855LkFTzkBJW7tnzOXt7dO5I0u0P2xLhUOIN2vTTsRTxeD4HhBbjJHZNbeu4I%2BX78%2FsEbHVTPXTgmIh3zA%2FHkdWoF6GFujMivr5iNcNs0mNnxz2PXCWK%2B4PGgnUxswydE5fB3P%2FDcl4Qw5OxklMnPY8u2T13b9cLbimOZ%2FEsuo784roPi%2B6ROjj64PX5F5AVoleOJLQPKJ1TG0wO5MxR4b%2Fg62oukD%2FX0J2apN8pJMd%2B8rtmOp7w0TbGcFLcZMadPkjutqYm%2FxAR9B3E5hrgFhORKELIUNfAvSvjpgiQyl6WBxNJA5AAaqDXQ4PTimz%2Byv%2F269%2BzL%2B6Vx%2BV26%2FnKkScyYIxMLqPjQ9YKpO3Edwz5ft56MFt4ymV1ZEq0v%2BOK68%2FiUf1EQvMVTzFgEbpZseEC9t3%2FI9UOZHj7F3YUHZ6vM0Rs9WllB6jJ89ER7xN%2FXF5EDes17qIgHxV14I1QwenTBMLwJCgrOU2POJkOb4ieWTzxkG4G1zK4NuxBdvv4l61fyytH9t7NL90yVXtUjXq5A9M7onOG2mgmoiSwF4SHju6Jg4XPXJjvfN66MZNwo8QQqYN6oIGMlnqDghUpghV6C0XYZvmAq3pp%2F%2Fbxy%2Fn1wfPf055%2F%2F3M6Pqgi9fed%2FcCA4lv1%2F2Oer5ZOi%2FrhRfiyMf77z1o%2FvR7xaN%2FvHl351LSdYs4vIaQkr0DVSyXFB9LDxhTlGSJ6kOm%2BwqOQYkxKjbQuDdpu8Y%2BBZkwnyME0E7hMaTob47%2BUVecJRYLnO587gaFOTmFdETBVu%2FdlAoQyS1JpBkiDNqKBod0JnJuZ%2BoBhw9PSSKEaQGxECx55nvKVOmJOZ7W%2BWEZImMjJCyumW%2BUv4Oi4RlOJLFIERXrlL8JfobWuVWDojsR5GUzTD40mYJMDfzdCqQcwT1or05vv4THzkubOIXMjw0cGJLUlgSCi3qPOCOFJQKkiqfuhbPrG8HBOT1yCxaYRtFxYZg%2FgKk54xIgxgjaLG%2BBQ%2BLfBS4m4t%2FFoReDygtxXNrI4FnpyF0Dwv6YVyKH%2F%2BFkGnFZ2%2Bs8QaPbjXs9XN03I%2Bmb0sTu8ej%2B%2Bu6bj2A3%2FzW%2Fk%2BL72yC38P%2BFlm%2BRkcd6Vufi4rC8Gn4QVm4bp1Qc74YjwjO0tNw7YmDhEzeAiRhxvIukAg%2BnH8w8wyzYhxkG%2F9Mp7D%2Fsjgx9MNdy6fDOQzkBxFXBt7AuIeB4n9vfzCdMThdUnls9MuOtpVf8r0mQg62oE7HvuoEa2J3ilFy3tERgS3fb1PsIZizAhucJ79eTjuio0H%2FOQ5Q3Hl58INW13PRN7RKLLPHoede5%2BOjtLtn8NeuNQ53uT5EyYIfguOl8XoCxedRjvG3ybk7%2F3JpgfCQ%2FBMzwpx0%2FrimWE56a4ODDOFToMCVY%2Bn7rU0gpIA401zup7ap8WjCfWuJoMPaMgBVorCJaV5g486lHI8pnK5bqK3asziQ5%2B6S%2BtBzCb8oDyIqMqSLYAP0Hwga%2B2wlKzmAK0it2tBFFhc84RYJ3UnwKZwAtSCbDRZ0zLjL9WCbLJoiSLZFoCNzBDzb0wi0yCm3nAdxMtgEUDgEpTjUUjx7BnOiFz16novQPTC1J09L%2Fzt%2BCEbi4DBxIUxs2wyRFfIXiLCNQDKYBgMgBr4lpYzwUfK%2BuhbKNiOpAHraR%2BPhdFoAHjaTeVZkZUG8Yqsahm8kgPVogYYfCAHu1YDXAEXXFYWAOzTP1jMwGCWi%2FOP%2FcHVtXG1IDOWZFnvmpPFTpwsVVFOmj%2BorpUiYSQtKppvmgToEoumiuRK%2B6acoqdOO01nyCFuh9DXQDwRz2jsemhA3Q1kIh6a8qwz%2FgZFVtlZ25TDoUjhS1HmBmHWomaZnHUjtfREQj5GM9BycKgmEFkphhRABEu7JO3G211FRa3FQ16zgBVYAQuqqzSooXELCLvW63qOdTYYQFhfjshKoHzoaH3KL8ycrCG4N8pv4Wzqs%2FKbM%2Bs3ofzCYyPsE%2BTrG5QDJE0ReGpa0LDREDwvlhMObF%2BcXOgm4PImt%2FpMuDCjsvGwfXElFk6sWoSOnEvsqEXkULtXo67EwljTTGxz3vhwv1YiuNepayPQ0IaHN0BOki60Bq0mWgKgd6tp48OkUd2kIfFDabPEEIBMGF6UWVTdmHmD1%2FZxrauMxmte6%2Bik3brYiW05FhUGVmv5fILSsFpllk6tOZ8SPL5skGfPYbXWf1h9lKVpe7ha6FSDfw8%2B7jhpZUdZ0xKy1hTGWp9fpkoD6%2BpiqzbmZDX4XmNpoU4FvhEsTZ%2BkfSwtVgk376%2FPRte5poWLCDhgHu8F9fbpfuJw%2FuLx15%2BPj8atQzvsiQNGZD0wpx4yAqIXOeh1kMry4BJH7SfG3E8byMgPEgfyWVpXomQo9htHHWQ7TfsPPg%2FWuXPrrpbCkBvy62Ny1mI%2BiN0UJpYYUdbdyJ3NLECtY94n0vP66L3IaWamjDRTgjQzTXgWlSY1M55j0K0oANqYEttY0tpYPjezPn7eqyjPSoKr3cy%2FQmVs34Qd69Q8EGEXZ%2BSlLFofkq26ZBOVcpKN6vJpyZbX76swLziV9sql0qHAAtMngXId7UTHbIpeLwoVkJtzgoCDs1dVLaozRtECtbWOi9jVQlb01Kl1LIy22YdoUHLJjWta47fwuUh5DPIlTDwnhCVMjjwS%2BPXpj4EgWeYfZH0zwpgw0x0tZpFfJn3e%2BuZxN%2BGzDofD7BpJdIG9GKJyPqQ9j5YSuGy2hQQssAKnD2mWcd1rbFFdrtS8uibB1sGUxr0Ras664N4sT9QRfccdKGPJgjjMWf15nnql08zFc0MOKDbTWMk8fn9LOPShZB6gcnZbckpmY%2BYq%2Bgl3qF713poQwFNTM%2FM7ni53yc7FHgopnvYn5JV0%2FIeI1jAlZow1%2BUUYKW7N5p67RETYdlfjtbGFk40Wh2IgoMjixtZNsUombD9EW81iSiib1Uoj6DquEQPUnqK5b5vLxGy7pBl5IHxY7%2FqcVtEf853MrnGtmu%2FAfGAgn8ZwFmxlyANbrUQgehgs%2B9hQIgwse6vUMW5ouUoOUlE1%2FUbidOXqr%2FOn8LFTM%2FArKXI4SOUaGsFemog2Vwo4MGkiQdgXkCY8JE34oaI2xVts6Pnp1fHt5fmXvy7J%2FUzTivL2%2B7E2Ix6vziq0NuuKKhqNhnOzFFTbLK4Jkk%2FqXn2pW0ZrZdWQ2ktV7jaRNGYixaD%2B%2FzPDscbID4b%2F%2BpG6b4VgeTE3sQrQHXDqz9QCytb2YGoJBze1ypa9lmrX8HebWmwmwFdr9DKg7o218oxZzo%2BqzESOC8sxraVlYmVlkPJ1xGcdnoENmEda2dz9xixsEote43pAcAZbLB3Hrjczgt%2BCRG1qlXCJ7iqJYP0wgtZUortmSSsDIAbeZKdfklZmQcwZFq5kpvZPudxax%2FZAVUlFZt3EElT1rVW4pO6vDKl58iuAlalQ9%2B7J5Fer1KjvBwV7YFssT%2FXai8Lutq9alcTMpqg%2BeJdFuSsKKkKvKKiwADvxLqYK0cPOuV6ZhoHqFDs9X%2F7Ge1z6rT%2FGGiDFIKm41R36EHskxTpFH3Tj1a3RZrVXpdiNgAIjxO7R0gqDI4gQOzAMv6XiogA5%2FdudUApDD4YIE0yF%2BftGaPeMdSBEQgaUnySicZrZSr4pN5rCxl%2BXHa7C4d9hvDocDHYuF0xgbvsEtknVhRNj9DIJL8vtMl%2B82Tmd%2BPJg8%2Bbnm7ZRn3iGaWFCnVkeCjc6xb8jww9AwVDMF6X5WxvmNnMQ2Hq6gswP6bZ6mQBwsSmKssXmzlfGbG6zWyji1wyy1MvSJyZaesjjpvKFNCB2yTJUMceUrZRdP5FTRFTkoSwpAq9Jmirqkg5U%2FmpMuits7YThcIgmxhHu3zCNwDii3pKLwEOI3K1E8OF4rHFQ8OF7gDYco3jh0%2F09L8L8kl2kSYr6epMzmc%2Fa6ACMLIgCS3ShMaKzgXytEj2JH90nsic4qCrZRUB8t0v2wyqIE4vmLQVxCmm53RJUNhSeytH2laiiVN9MpO4LMQP5UXbJaGo4kzBccG%2BSbrfVKTXCF5sv%2FNSWz65nTazuQtZYbqwpAE1Qh7mda2Ue0BMVYcgD4LCmKEKQ8TopJLqH28kV%2BoC3OouFlvIDVSm3nZyar8dXX%2BlP8EV7XPmziP37XPgza2BqpO5nIXMDpa1Na0ml%2Bob95bL4MQnzKh9W7s8NB17PEtUwvaZZjhVYhg0uUPEWX8m9o64Z%2FJp6KXYB%2BiieXT7IQSFBDhsNpDIUM9XU3mAwY%2B9V7Zma6kfCI1G2Ek1nHofCx97kcYhDSPcQMx8Y7oW2C2ozOhzmnhI27q4wMAW65TDw1tjIHsNkMF%2B19rwQGFFF%2BUAbVyhe0kthZhZ981le57m8aWhDcY53J9ArOZiv8MXp83y18zfuMaBlft859R7WmVhdoi%2BlvQvlSi26hK7li3srtSgTWT5ISm63oE30qRBRRhb2LP25CH9ttWp2lv0Mz%2BAepby%2Fk%2BadJyeUZQN4snUWnlz02FCU4%2BXVYFNtpQ%2FAXr7uiYG0MajtKyMNPY%2BbhvhKaqusHNLgxM7xPrRF20a6HUpMS03BbqIwVHh9%2FckQV9faDXCBycsmPv2eAS41VrJUN1IcMN61FA0DE5%2FVEX7jwIj%2BrAqyxDOJbtBK0Fg8BYxHWY%2FGb8wtLdJe11sMpYFJX6Xayr6H0uykawhlfdt87Va73SjNwr2vnjtCPnEEjC3HiuJN%2BgHcdfWMU8FkoMYLFkK5yEqbucijB%2Fd6trp5Ws4ns5fF6d3j8d11N9tCNBaOwm%2BZ59WnLDh4CjtjwfPoBuRN29klrAvk6zAKemMBKeC7sraH3gSkFE2A3zwgBWbbPuU7v2%2Bis6tC81Mf9LDxnVmIYZKynvy%2B%2BHgKWbCe6SkpWR8PpfauO0XL2V5z3szmJiibeHjueRjQYOxANJh4Y0zyH1qN7IWPX9GOUmvCsLFka5P1TiV5lPYRhlXeMMsXOrmhpJbGwrDguVQF6%2FXFO9SwOKcIrb8Ov8LH3lwtPwzPCuXAPK6DvHmy73nwk8Rl4yC5rFhuNFMcH3ouMUatxTsesumNa5JF9Pw%2F%3C%2Fdiagram%3E%3Cdiagram%20id%3D%22YkMsiQFkmj2jwWWi3ArU%22%20name%3D%22GitHub%20management%22%3E7Vxbc%2BI2FP41zLQPML5jHgOE7UN2u5Pdbbd9k23ZqDGWa4sA%2FfWVLPluHC4GkwzJZLCPLpZ1zvn0nSORgTpbbT9FIFx%2Bxg70B4rkbAfqfKAosjI26AeT7LhEV3Uu8CLkiEq54Bv6DwqhJKRr5MC4VJFg7BMUloU2DgJok5IMRBHelKu52C8%2FNQQerAm%2B2cCvS%2F9EDllyqamMc%2FlvEHnL9MmyMeElK5BWFm8SL4GDNwWR%2BjhQZxHGhF%2BttjPos8lL54W3W%2BwpzQYWwYAc0kANv7v6wx%2Fbxy%2Bff4%2BnX17ms7%2Btoazwbl6BvxZvPFAMn3Y4tXxsv%2Fy7xgSywZOdmBGDiWjxCkQeCgbqAy2VCn%2BaFFLlT%2BmVhSMHRrxGgAPIpSFwHBR4oqGom3ZKrzzx6fPPRXEY5cKbH2AcgqBxaC4OyDBObJ09RJHfeIpVEIBVSC8CKw75cH%2BEDiBsvGndqGmMDTI%2ButqDGobL52loYx%2BL2XoF0S%2FDYVH%2BKx%2FNEW9WfZHPAAXNQ1ZKo1IivA4cyKxaosWbJSLwWwhsVrqhIERlS7Ly6Z1ML31gQX8K7BcvaTbj7zAX6mbDFZgjS%2BI%2BrTJQVP7D5Mj3U7kDXbDm%2BiURfoGVLoGPPGp1cx%2B6rI4XAQfBUq%2BPBvstlM1RRKELYdYMgjh%2F6aJzC39%2FhRGB24JIOPsniFeQRDtaJS0VuCOAVzHF%2FSaHMVkaC%2BGygGGGkAEBnV7WdY4u9EIAzDFgo97B5nIDbHbdzPKL7osCRBDwz3bZL3DTBjJ1KGoe5BkjmC1B4FF%2B8A5wg8pdnf2ejhyLxUKbqZeBB6l%2FeNDu8PCh4OEZ%2BnQ5gz1DxNcI2zB%2BFxDxTqmFrPSOHWojdgjjEMwyMxXJikBgL1u0L7%2Bt%2FaJ%2B9TrUOzo0Ha2mSVpiKpZqGGkP4ukN0dzxWpnoZbU0aEVpUIp2KaVoem2KoUOja3GLI7LEHg6A%2F5hLp2UXzOs8YRyKqf8HErITcw%2FWBJcVQycw2v1k7Ud6evuX6C65mW9Ldztxt1cDMV5HNmwzPvGehKI8JG0TIggwm4VWjUbQp0HdazkH0aQf0fQrRnTQmSWoZtkSshxE2gUfqWiVa%2FkhisCuUC1kFeL9z8n8ObW4SSUFUamvTVrr0ws%2Bgtzksjk53QrThEwBGmY%2BA9Qm23ximF62pxR1bWoXbIWeMo9ENvAfRMEKOQ43XUhXKGAl%2FTGLEtNHO9enA33eaGOtjiOSWqLLLMNzBCZII0nXJ2Vj4HenmtiusQF23RiSQRUxOlBf6lwF9Y1GI%2BiBIe0ZOICAYWwv4QrQ3kkEIRuAWOoZwCazmGJvvgSormtKjIK9mXJZ7U1LZDL%2BnFS84MOhbrOgAz2PYhQWmUlHS7decUGtvkpkK0dxlahCSHdLt7GX9rcpUEp%2B6gp8hiH1Q5jglkSWnI9TS7fpxY%2Fnp0QIWK%2BIVVjHVBWM5EZ8BIgl03CQ1KKa8ihJkKYIixJWa4%2FWK1qm2iBlVTYyuiJpqJC8t%2BGmyXbK1nUWQ10sEg%2FpxuqM8oI0bqAmZoPRqRczOrOFL87h67Xpomva0Lab6KJl6pqeqqdLuqipFSTomy6qk37pYoks5txxD10sKlSue4%2BAp1MUzUvS3SblLGqaKqsvanoso1T1CnNVr8EQxzUw%2BPp8w%2FQwdZTz6eFQGmU6LBO7U%2Bnh5QlhatIl6K7qyvdRGO9bJIvai0O%2Ba%2ByiLUOR2qL4xhJacW7XVZqd2zEsQze6WUyVyZ7orkjhrsngUkr5PoAbbhH5mfZBrwut6F3eiN1kYN8viKuHgrh5EyBuKGVmUQHxY%2BtfBvS1%2Bmbk3rjS8rHVdVzpJCTzo4SVezNOvYWV2v7dpFPCyhltA1CQB5V2uvMoWZBsIAy4W7ENDy6kMSVMLpLDGdIKUudlsSbFMaXwgFUlOX2PL0%2BJZaQygvQeX2r1pEa2G3aVeBI4Y9C8QmmGLmkXiCcN49a2H%2Bq0%2FhwEYPuOMfVlv6hCkTmCILmJUhXPODtPEWPDM04blAzagnlV5%2B7wpxmbemMJJd3olwMft9fVM581D%2BSzaTKgZz47VirkZtzOZ5VqEmN8jSRGfZ%2FkppMYqcN0ksSQNPXm8xZGz1HytXbDD%2FfuzqPV8%2BhCfUvgqhuNr8pIHkkfKCashfW9x4S6XFPx%2BznAd%2FMDbBzX%2FYThGT55P2EooEM1Gvj9dU8Y6koNO56zGE76DrwWVXdwnBBA021k6IZtQsu9QDw%2FkW4sntf3f33k7Hg%2B0V9LMC9kIqbnGT1xuASsIH%2FbtqyelCQAeAuCkhbYZeLEZBLyW3jiPSVwgr2a4xtLCRgN%2B5K3S%2FjLAb58VgigH3og1pBvKgTQ72cNL3nWUG34jtGVA4B6Svge450FupUT4GrDt0yvrOKeT5GdlYg9gOZ1mYjVD07VjLvG6U4SsarenoidGK31L5SIrW88Z2Hq7WZjuzxSJuuySO7ebjZWv2f7Lpvt63%2BxN%2BrZvjub65DN9b7UG0cc4bqQD0sfyoerbO6SXw6it%2Fn%2FPuKonv8HKfXxfw%3D%3D%3C%2Fdiagram%3E%3C%2Fmxfile%3E)
- [GitHub diagram](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=YkMsiQFkmj2jwWWi3ArU&title=EE-2412%20-%20JSON%20Schema%20release#R%3Cmxfile%20pages%3D%222%22%3E%3Cdiagram%20id%3D%22C5RBs43oDa-KdzZeNtuy%22%20name%3D%22Release%20workflow%22%3E7V1bd5u4Fv41XtM%2BxIv75TH3ZE6TaZJ2MunLWcTINhMMLmDH6a8fCRAGtI0J5mY37lqNkUGA9tbWt68aiKez1aVnzKc3ronsgcCZq4F4NhAEXuZ4%2FIe0vEUtiqBHDRPPMuOT1g0P1i8UN3Jx68IykZ85MXBdO7Dm2caR6zhoFGTaDM9zX7OnjV07e9e5MUFMw8PIsNnWR8sMplGrJqjr9itkTab0zrwSv9%2FMoCfHb%2BJPDdN9TTWJ5wPx1HPdIPo2W50imwweHZfH67dH%2B8uLcvnnnf%2FT%2BH7yv2%2B3fx9FnV2855LkFTzkBJW7tnzOXt7dO5I0u0P2xLhUOIN2vTTsRTxeD4HhBbjJHZNbeu4I%2BX78%2FsEbHVTPXTgmIh3zA%2FHkdWoF6GFujMivr5iNcNs0mNnxz2PXCWK%2B4PGgnUxswydE5fB3P%2FDcl4Qw5OxklMnPY8u2T13b9cLbimOZ%2FEsuo784roPi%2B6ROjj64PX5F5AVoleOJLQPKJ1TG0wO5MxR4b%2Fg62oukD%2FX0J2apN8pJMd%2B8rtmOp7w0TbGcFLcZMadPkjutqYm%2FxAR9B3E5hrgFhORKELIUNfAvSvjpgiQyl6WBxNJA5AAaqDXQ4PTimz%2Byv%2F269%2BzL%2B6Vx%2BV26%2FnKkScyYIxMLqPjQ9YKpO3Edwz5ft56MFt4ymV1ZEq0v%2BOK68%2FiUf1EQvMVTzFgEbpZseEC9t3%2FI9UOZHj7F3YUHZ6vM0Rs9WllB6jJ89ER7xN%2FXF5EDes17qIgHxV14I1QwenTBMLwJCgrOU2POJkOb4ieWTzxkG4G1zK4NuxBdvv4l61fyytH9t7NL90yVXtUjXq5A9M7onOG2mgmoiSwF4SHju6Jg4XPXJjvfN66MZNwo8QQqYN6oIGMlnqDghUpghV6C0XYZvmAq3pp%2F%2Fbxy%2Fn1wfPf055%2F%2F3M6Pqgi9fed%2FcCA4lv1%2F2Oer5ZOi%2FrhRfiyMf77z1o%2FvR7xaN%2FvHl351LSdYs4vIaQkr0DVSyXFB9LDxhTlGSJ6kOm%2BwqOQYkxKjbQuDdpu8Y%2BBZkwnyME0E7hMaTob47%2BUVecJRYLnO587gaFOTmFdETBVu%2FdlAoQyS1JpBkiDNqKBod0JnJuZ%2BoBhw9PSSKEaQGxECx55nvKVOmJOZ7W%2BWEZImMjJCyumW%2BUv4Oi4RlOJLFIERXrlL8JfobWuVWDojsR5GUzTD40mYJMDfzdCqQcwT1or05vv4THzkubOIXMjw0cGJLUlgSCi3qPOCOFJQKkiqfuhbPrG8HBOT1yCxaYRtFxYZg%2FgKk54xIgxgjaLG%2BBQ%2BLfBS4m4t%2FFoReDygtxXNrI4FnpyF0Dwv6YVyKH%2F%2BFkGnFZ2%2Bs8QaPbjXs9XN03I%2Bmb0sTu8ej%2B%2Bu6bj2A3%2FzW%2Fk%2BL72yC38P%2BFlm%2BRkcd6Vufi4rC8Gn4QVm4bp1Qc74YjwjO0tNw7YmDhEzeAiRhxvIukAg%2BnH8w8wyzYhxkG%2F9Mp7D%2Fsjgx9MNdy6fDOQzkBxFXBt7AuIeB4n9vfzCdMThdUnls9MuOtpVf8r0mQg62oE7HvuoEa2J3ilFy3tERgS3fb1PsIZizAhucJ79eTjuio0H%2FOQ5Q3Hl58INW13PRN7RKLLPHoede5%2BOjtLtn8NeuNQ53uT5EyYIfguOl8XoCxedRjvG3ybk7%2F3JpgfCQ%2FBMzwpx0%2FrimWE56a4ODDOFToMCVY%2Bn7rU0gpIA401zup7ap8WjCfWuJoMPaMgBVorCJaV5g486lHI8pnK5bqK3asziQ5%2B6S%2BtBzCb8oDyIqMqSLYAP0Hwga%2B2wlKzmAK0it2tBFFhc84RYJ3UnwKZwAtSCbDRZ0zLjL9WCbLJoiSLZFoCNzBDzb0wi0yCm3nAdxMtgEUDgEpTjUUjx7BnOiFz16novQPTC1J09L%2Fzt%2BCEbi4DBxIUxs2wyRFfIXiLCNQDKYBgMgBr4lpYzwUfK%2BuhbKNiOpAHraR%2BPhdFoAHjaTeVZkZUG8Yqsahm8kgPVogYYfCAHu1YDXAEXXFYWAOzTP1jMwGCWi%2FOP%2FcHVtXG1IDOWZFnvmpPFTpwsVVFOmj%2BorpUiYSQtKppvmgToEoumiuRK%2B6acoqdOO01nyCFuh9DXQDwRz2jsemhA3Q1kIh6a8qwz%2FgZFVtlZ25TDoUjhS1HmBmHWomaZnHUjtfREQj5GM9BycKgmEFkphhRABEu7JO3G211FRa3FQ16zgBVYAQuqqzSooXELCLvW63qOdTYYQFhfjshKoHzoaH3KL8ycrCG4N8pv4Wzqs%2FKbM%2Bs3ofzCYyPsE%2BTrG5QDJE0ReGpa0LDREDwvlhMObF%2BcXOgm4PImt%2FpMuDCjsvGwfXElFk6sWoSOnEvsqEXkULtXo67EwljTTGxz3vhwv1YiuNepayPQ0IaHN0BOki60Bq0mWgKgd6tp48OkUd2kIfFDabPEEIBMGF6UWVTdmHmD1%2FZxrauMxmte6%2Bik3brYiW05FhUGVmv5fILSsFpllk6tOZ8SPL5skGfPYbXWf1h9lKVpe7ha6FSDfw8%2B7jhpZUdZ0xKy1hTGWp9fpkoD6%2BpiqzbmZDX4XmNpoU4FvhEsTZ%2BkfSwtVgk376%2FPRte5poWLCDhgHu8F9fbpfuJw%2FuLx15%2BPj8atQzvsiQNGZD0wpx4yAqIXOeh1kMry4BJH7SfG3E8byMgPEgfyWVpXomQo9htHHWQ7TfsPPg%2FWuXPrrpbCkBvy62Ny1mI%2BiN0UJpYYUdbdyJ3NLECtY94n0vP66L3IaWamjDRTgjQzTXgWlSY1M55j0K0oANqYEttY0tpYPjezPn7eqyjPSoKr3cy%2FQmVs34Qd69Q8EGEXZ%2BSlLFofkq26ZBOVcpKN6vJpyZbX76swLziV9sql0qHAAtMngXId7UTHbIpeLwoVkJtzgoCDs1dVLaozRtECtbWOi9jVQlb01Kl1LIy22YdoUHLJjWta47fwuUh5DPIlTDwnhCVMjjwS%2BPXpj4EgWeYfZH0zwpgw0x0tZpFfJn3e%2BuZxN%2BGzDofD7BpJdIG9GKJyPqQ9j5YSuGy2hQQssAKnD2mWcd1rbFFdrtS8uibB1sGUxr0Ras664N4sT9QRfccdKGPJgjjMWf15nnql08zFc0MOKDbTWMk8fn9LOPShZB6gcnZbckpmY%2BYq%2Bgl3qF713poQwFNTM%2FM7ni53yc7FHgopnvYn5JV0%2FIeI1jAlZow1%2BUUYKW7N5p67RETYdlfjtbGFk40Wh2IgoMjixtZNsUombD9EW81iSiib1Uoj6DquEQPUnqK5b5vLxGy7pBl5IHxY7%2FqcVtEf853MrnGtmu%2FAfGAgn8ZwFmxlyANbrUQgehgs%2B9hQIgwse6vUMW5ouUoOUlE1%2FUbidOXqr%2FOn8LFTM%2FArKXI4SOUaGsFemog2Vwo4MGkiQdgXkCY8JE34oaI2xVts6Pnp1fHt5fmXvy7J%2FUzTivL2%2B7E2Ix6vziq0NuuKKhqNhnOzFFTbLK4Jkk%2FqXn2pW0ZrZdWQ2ktV7jaRNGYixaD%2B%2FzPDscbID4b%2F%2BpG6b4VgeTE3sQrQHXDqz9QCytb2YGoJBze1ypa9lmrX8HebWmwmwFdr9DKg7o218oxZzo%2BqzESOC8sxraVlYmVlkPJ1xGcdnoENmEda2dz9xixsEote43pAcAZbLB3Hrjczgt%2BCRG1qlXCJ7iqJYP0wgtZUortmSSsDIAbeZKdfklZmQcwZFq5kpvZPudxax%2FZAVUlFZt3EElT1rVW4pO6vDKl58iuAlalQ9%2B7J5Fer1KjvBwV7YFssT%2FXai8Lutq9alcTMpqg%2BeJdFuSsKKkKvKKiwADvxLqYK0cPOuV6ZhoHqFDs9X%2F7Ge1z6rT%2FGGiDFIKm41R36EHskxTpFH3Tj1a3RZrVXpdiNgAIjxO7R0gqDI4gQOzAMv6XiogA5%2FdudUApDD4YIE0yF%2BftGaPeMdSBEQgaUnySicZrZSr4pN5rCxl%2BXHa7C4d9hvDocDHYuF0xgbvsEtknVhRNj9DIJL8vtMl%2B82Tmd%2BPJg8%2Bbnm7ZRn3iGaWFCnVkeCjc6xb8jww9AwVDMF6X5WxvmNnMQ2Hq6gswP6bZ6mQBwsSmKssXmzlfGbG6zWyji1wyy1MvSJyZaesjjpvKFNCB2yTJUMceUrZRdP5FTRFTkoSwpAq9Jmirqkg5U%2FmpMuits7YThcIgmxhHu3zCNwDii3pKLwEOI3K1E8OF4rHFQ8OF7gDYco3jh0%2F09L8L8kl2kSYr6epMzmc%2Fa6ACMLIgCS3ShMaKzgXytEj2JH90nsic4qCrZRUB8t0v2wyqIE4vmLQVxCmm53RJUNhSeytH2laiiVN9MpO4LMQP5UXbJaGo4kzBccG%2BSbrfVKTXCF5sv%2FNSWz65nTazuQtZYbqwpAE1Qh7mda2Ue0BMVYcgD4LCmKEKQ8TopJLqH28kV%2BoC3OouFlvIDVSm3nZyar8dXX%2BlP8EV7XPmziP37XPgza2BqpO5nIXMDpa1Na0ml%2Bob95bL4MQnzKh9W7s8NB17PEtUwvaZZjhVYhg0uUPEWX8m9o64Z%2FJp6KXYB%2BiieXT7IQSFBDhsNpDIUM9XU3mAwY%2B9V7Zma6kfCI1G2Ek1nHofCx97kcYhDSPcQMx8Y7oW2C2ozOhzmnhI27q4wMAW65TDw1tjIHsNkMF%2B19rwQGFFF%2BUAbVyhe0kthZhZ981le57m8aWhDcY53J9ArOZiv8MXp83y18zfuMaBlft859R7WmVhdoi%2BlvQvlSi26hK7li3srtSgTWT5ISm63oE30qRBRRhb2LP25CH9ttWp2lv0Mz%2BAepby%2Fk%2BadJyeUZQN4snUWnlz02FCU4%2BXVYFNtpQ%2FAXr7uiYG0MajtKyMNPY%2BbhvhKaqusHNLgxM7xPrRF20a6HUpMS03BbqIwVHh9%2FckQV9faDXCBycsmPv2eAS41VrJUN1IcMN61FA0DE5%2FVEX7jwIj%2BrAqyxDOJbtBK0Fg8BYxHWY%2FGb8wtLdJe11sMpYFJX6Xayr6H0uykawhlfdt87Va73SjNwr2vnjtCPnEEjC3HiuJN%2BgHcdfWMU8FkoMYLFkK5yEqbucijB%2Fd6trp5Ws4ns5fF6d3j8d11N9tCNBaOwm%2BZ59WnLDh4CjtjwfPoBuRN29klrAvk6zAKemMBKeC7sraH3gSkFE2A3zwgBWbbPuU7v2%2Bis6tC81Mf9LDxnVmIYZKynvy%2B%2BHgKWbCe6SkpWR8PpfauO0XL2V5z3szmJiibeHjueRjQYOxANJh4Y0zyH1qN7IWPX9GOUmvCsLFka5P1TiV5lPYRhlXeMMsXOrmhpJbGwrDguVQF6%2FXFO9SwOKcIrb8Ov8LH3lwtPwzPCuXAPK6DvHmy73nwk8Rl4yC5rFhuNFMcH3ouMUatxTsesumNa5JF9Pw%2F%3C%2Fdiagram%3E%3Cdiagram%20id%3D%22YkMsiQFkmj2jwWWi3ArU%22%20name%3D%22GitHub%20management%22%3E7Vxbc%2BI2FP41zLQPML5jHgOE7UN2u5Pdbbd9k23ZqDGWa4sA%2FfWVLPluHC4GkwzJZLCPLpZ1zvn0nSORgTpbbT9FIFx%2Bxg70B4rkbAfqfKAosjI26AeT7LhEV3Uu8CLkiEq54Bv6DwqhJKRr5MC4VJFg7BMUloU2DgJok5IMRBHelKu52C8%2FNQQerAm%2B2cCvS%2F9EDllyqamMc%2FlvEHnL9MmyMeElK5BWFm8SL4GDNwWR%2BjhQZxHGhF%2BttjPos8lL54W3W%2BwpzQYWwYAc0kANv7v6wx%2Fbxy%2Bff4%2BnX17ms7%2Btoazwbl6BvxZvPFAMn3Y4tXxsv%2Fy7xgSywZOdmBGDiWjxCkQeCgbqAy2VCn%2BaFFLlT%2BmVhSMHRrxGgAPIpSFwHBR4oqGom3ZKrzzx6fPPRXEY5cKbH2AcgqBxaC4OyDBObJ09RJHfeIpVEIBVSC8CKw75cH%2BEDiBsvGndqGmMDTI%2ButqDGobL52loYx%2BL2XoF0S%2FDYVH%2BKx%2FNEW9WfZHPAAXNQ1ZKo1IivA4cyKxaosWbJSLwWwhsVrqhIERlS7Ly6Z1ML31gQX8K7BcvaTbj7zAX6mbDFZgjS%2BI%2BrTJQVP7D5Mj3U7kDXbDm%2BiURfoGVLoGPPGp1cx%2B6rI4XAQfBUq%2BPBvstlM1RRKELYdYMgjh%2F6aJzC39%2FhRGB24JIOPsniFeQRDtaJS0VuCOAVzHF%2FSaHMVkaC%2BGygGGGkAEBnV7WdY4u9EIAzDFgo97B5nIDbHbdzPKL7osCRBDwz3bZL3DTBjJ1KGoe5BkjmC1B4FF%2B8A5wg8pdnf2ejhyLxUKbqZeBB6l%2FeNDu8PCh4OEZ%2BnQ5gz1DxNcI2zB%2BFxDxTqmFrPSOHWojdgjjEMwyMxXJikBgL1u0L7%2Bt%2FaJ%2B9TrUOzo0Ha2mSVpiKpZqGGkP4ukN0dzxWpnoZbU0aEVpUIp2KaVoem2KoUOja3GLI7LEHg6A%2F5hLp2UXzOs8YRyKqf8HErITcw%2FWBJcVQycw2v1k7Ud6evuX6C65mW9Ldztxt1cDMV5HNmwzPvGehKI8JG0TIggwm4VWjUbQp0HdazkH0aQf0fQrRnTQmSWoZtkSshxE2gUfqWiVa%2FkhisCuUC1kFeL9z8n8ObW4SSUFUamvTVrr0ws%2Bgtzksjk53QrThEwBGmY%2BA9Qm23ximF62pxR1bWoXbIWeMo9ENvAfRMEKOQ43XUhXKGAl%2FTGLEtNHO9enA33eaGOtjiOSWqLLLMNzBCZII0nXJ2Vj4HenmtiusQF23RiSQRUxOlBf6lwF9Y1GI%2BiBIe0ZOICAYWwv4QrQ3kkEIRuAWOoZwCazmGJvvgSormtKjIK9mXJZ7U1LZDL%2BnFS84MOhbrOgAz2PYhQWmUlHS7decUGtvkpkK0dxlahCSHdLt7GX9rcpUEp%2B6gp8hiH1Q5jglkSWnI9TS7fpxY%2Fnp0QIWK%2BIVVjHVBWM5EZ8BIgl03CQ1KKa8ihJkKYIixJWa4%2FWK1qm2iBlVTYyuiJpqJC8t%2BGmyXbK1nUWQ10sEg%2FpxuqM8oI0bqAmZoPRqRczOrOFL87h67Xpomva0Lab6KJl6pqeqqdLuqipFSTomy6qk37pYoks5txxD10sKlSue4%2BAp1MUzUvS3SblLGqaKqsvanoso1T1CnNVr8EQxzUw%2BPp8w%2FQwdZTz6eFQGmU6LBO7U%2Bnh5QlhatIl6K7qyvdRGO9bJIvai0O%2Ba%2ByiLUOR2qL4xhJacW7XVZqd2zEsQze6WUyVyZ7orkjhrsngUkr5PoAbbhH5mfZBrwut6F3eiN1kYN8viKuHgrh5EyBuKGVmUQHxY%2BtfBvS1%2Bmbk3rjS8rHVdVzpJCTzo4SVezNOvYWV2v7dpFPCyhltA1CQB5V2uvMoWZBsIAy4W7ENDy6kMSVMLpLDGdIKUudlsSbFMaXwgFUlOX2PL0%2BJZaQygvQeX2r1pEa2G3aVeBI4Y9C8QmmGLmkXiCcN49a2H%2Bq0%2FhwEYPuOMfVlv6hCkTmCILmJUhXPODtPEWPDM04blAzagnlV5%2B7wpxmbemMJJd3olwMft9fVM581D%2BSzaTKgZz47VirkZtzOZ5VqEmN8jSRGfZ%2FkppMYqcN0ksSQNPXm8xZGz1HytXbDD%2FfuzqPV8%2BhCfUvgqhuNr8pIHkkfKCashfW9x4S6XFPx%2BznAd%2FMDbBzX%2FYThGT55P2EooEM1Gvj9dU8Y6koNO56zGE76DrwWVXdwnBBA021k6IZtQsu9QDw%2FkW4sntf3f33k7Hg%2B0V9LMC9kIqbnGT1xuASsIH%2FbtqyelCQAeAuCkhbYZeLEZBLyW3jiPSVwgr2a4xtLCRgN%2B5K3S%2FjLAb58VgigH3og1pBvKgTQ72cNL3nWUG34jtGVA4B6Svge450FupUT4GrDt0yvrOKeT5GdlYg9gOZ1mYjVD07VjLvG6U4SsarenoidGK31L5SIrW88Z2Hq7WZjuzxSJuuySO7ebjZWv2f7Lpvt63%2BxN%2BrZvjub65DN9b7UG0cc4bqQD0sfyoerbO6SXw6it%2Fn%2FPuKonv8HKfXxfw%3D%3D%3C%2Fdiagram%3E%3C%2Fmxfile%3E)